integrations/stripe/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@botpresshub/stripe",
```

Comment:
Does it need to be `@botpresshub` and should it be private ?

Reply:
I think its used for pnpm script selectors. I like the uniformity of it